MyRoutine/MyRoutine/Data/Source/DateService.swift

```diff
@@ -31,4 +31,18 @@ class DateService {
         }
         return result
     }
+    
+    func allDaysInWeek(dayInWeek: Date) -> [Date] {
+        let gregorian = Calendar(identifier: .gregorian)
+        guard let sunday = gregorian.date(from: gregorian.dateComponents([.yearForWeekOfYear, .weekOfYear],
+                                                                         from: dayInWeek)) else { return [] }
+        var week = [Date]()
+        for numberDayAdded in 2...8 {
+            guard let day = gregorian.date(byAdding: .day,
+                                           value: numberDayAdded,
+                                           to: sunday) else { return [] }
+            week.append(day.addingTimeInterval(-61200))
```

Comment:
Comment rõ hơn em nhé
```suggestion
            // - 17 hours ...
            week.append(day.addingTimeInterval(-61200))
```